Flush all open files after showing files
Original patch by Yauheni Kaliuta <yauheni.kaliuta@nokia.com>

--- a/coreutils/ls.c
+++ b/coreutils/ls.c
@@ -701,6 +701,7 @@ static void display_files(struct dnode *
 			}
 		}
 		putchar('\n');
+		fflush(NULL);
 		column = 0;
 	}
 }
